src/System.IO.FileSystem/src/System/IO/Enumeration/FileSystemEntry.Unix.cs

```diff
@@ -129,6 +129,7 @@ public FileAttributes Attributes
         public DateTimeOffset LastAccessTimeUtc => _status.GetLastAccessTime(FullPath, continueOnError: true);
         public DateTimeOffset LastWriteTimeUtc => _status.GetLastWriteTime(FullPath, continueOnError: true);
         public bool IsDirectory => _status.InitiallyDirectory;
+        public bool IsHidden => _directoryEntry.Name[0] == '.';
```

Comment:
What if Name is null or empty?  I see us doing checks elsewhere, like:
https://github.com/dotnet/corefx/pull/27318/files#diff-61f827c8c50fc3e07c9e00ab7bc18bf3R74
for whether the Length == 0 and like:
https://github.com/dotnet/corefx/pull/27318/files#diff-61f827c8c50fc3e07c9e00ab7bc18bf3R91
for whether Name == null.

Reply:
Good catch- totally missed my doubling that up